javascript/spread-operator.md

```diff
@@ -0,0 +1,29 @@
+### The spread operator
+
+Today I learned about a powerful feature of ES6 (ES6 being the latest standard of JavaScript implementation in the browsers), the so-called spread operator that looks like this `...`.
```

Comment:
One powerful feature of ES6, the at-present current JavaScript implementation, is the spread operator, represented by three periods '...'.